docs/user_guide.md

```diff
@@ -756,19 +769,24 @@ providing a quick overview on which checkers are available in the analyzers.
 
 ~~~~~~~~~~~~~~~~~~~~~
 usage: CodeChecker checkers [-h] [--analyzers ANALYZER [ANALYZER ...]]
-                            [--details] [--only-enabled | --only-disabled]
+                            [--details] [--profile {PROFILE/list}]
+                            [--only-enabled | --only-disabled]
                             [-o {rows,table,csv,json}]
                             [--verbose {info,debug,debug_analyzer}]
 
 Get the list of checkers available and their enabled status in the supported
 analyzers. Currently supported analyzers are: clangsa, clang-tidy.
 
 optional arguments:
-  -h, --help            show this help message and exit
+  -h, --help            Show this help message and exit
```

Comment:
Bogus change. I believe argparse creates the `--help` command with a lower-case help string.

Reply:
All the other argument descriptions start with an uppercase letter. This segment of the output of `CodeChecker checkers -h` currently looks like this:
```
optional arguments:
  -h, --help            show this help message and exit
  --analyzers ANALYZER [ANALYZER ...]
                        Show checkers only from the analyzers specified.
  --details             Show details about the checker, such as description,
                        if available.
```
Etc.
But this is a good reminder to put a comma at the end of the sentence too.